examples/adaptation.ipynb

```diff
@@ -0,0 +1,604 @@
+      "1      0.065512    0.065512          0.0    0.006782     -0.05873   \n",
+      "2      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "3      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "4      0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "..          ...         ...          ...         ...          ...   \n",
+      "225    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "226    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "227    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "228    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "229    0.000000    0.000000          0.0    0.000000      0.00000   \n",
+      "\n",
+      "         AO1_cost  AO1_bc_ratio      AO2_cost  AO2_bc_ratio  \n",
+      "0    1.893107e+05           0.0  3.663338e+05  0.000000e+00  \n",
+      "1    1.430648e+06           0.0  2.768437e+06 -2.121409e-08  \n",
+      "2    1.920151e+05           0.0  3.715672e+05  0.000000e+00  \n",
+      "3    9.735977e+04           0.0  1.884003e+05  0.000000e+00  \n",
+      "4    6.490651e+04           0.0  1.256002e+05  0.000000e+00  \n",
+      "..            ...           ...           ...           ...  \n",
+      "225  2.001284e+05           0.0  3.872672e+05  0.000000e+00  \n",
+      "226  2.055373e+05           0.0  3.977339e+05  0.000000e+00  \n",
+      "227  3.515769e+04           0.0  6.803343e+04  0.000000e+00  \n",
+      "228  2.055373e+05           0.0  3.977339e+05  0.000000e+00  \n",
+      "229  3.515769e+04           0.0  6.803343e+04  0.000000e+00  \n",
+      "\n",
+      "[230 rows x 9 columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "_network_config = NetworkConfigWrapper.from_data(None, _network_config_data)\n",
+    "valid_analysis_ini = root_dir.joinpath(\"analyses.ini\")\n",
+    "\n",
+    "\n",
+    "_analysis_config = AnalysisConfigWrapper.from_data_with_network(\n",
+    "        valid_analysis_ini, _analysis_data, _network_config\n",
+    "    )\n",
+    "\n",
+    "_analysis_input = AnalysisInputWrapper.from_input(\n",
+    "        analysis=_analysis_config.config_data.adaptation,\n",
+    "        analysis_config=_analysis_config,\n",
+    "        graph_file=_analysis_config.graph_files.base_network,\n",
+    "        graph_file_hazard=_analysis_config.graph_files.base_network_hazard,\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "\n",
+    "valid_adaptation_config = (_analysis_input, _analysis_config)\n",
+    "_adaptation = Adaptation(valid_adaptation_config[0], valid_adaptation_config[1])\n",
+    "\n",
+    "# 2. Run test.\n",
+    "_benefit = _adaptation.run_benefit()\n",
+    "\n",
+    "cost = _adaptation.run_cost()\n",
+    "\n",
+    "\n",
+    "_res_2 = _adaptation.calculate_bc_ratio(cost_gdf=cost, benefit_gdf=_benefit)\n",
```

Comment:
This should be enough:
```suggestion
    "_res_2 = _adaptation.execute()\n",
```

Reply:
This will be removed anyway when the handler is made accessible